framework-docs/src/docs/asciidoc/spring-framework.adocbook

```diff
@@ -1,6 +1,7 @@
 :noheader:
 :toc:
 include::attributes.adoc[]
+include::page-layout.adoc[]
```

Comment:
`page-layout.adoc` is specific to the HTML5 backend and as such should not be applied for the generated PDF.

For the PDF, we explicitly apply document configuration local to the `spring-framework.adocbook` file -- as can be seen by the existing `:noheader:` and `:toc:` entries.